client/blocks/plugins-update-manager/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button } from '@wordpress/components';
+import { plus } from '@wordpress/icons';
+import MainComponent from 'calypso/components/main';
+import NavigationHeader from 'calypso/components/navigation-header';
+import { ScheduleCreate } from './schedule-create';
+import { ScheduleList } from './schedule-list';
+
+interface Props {
+	siteSlug: string;
+	context: 'list' | 'create';
+	onNavBack?: () => void;
+	onCreateNewSchedule?: () => void;
+}
+export const PluginsUpdateManager = ( props: Props ) => {
+	const { context, onNavBack, onCreateNewSchedule } = props;
+
+	return (
+		<MainComponent wideLayout>
+			<NavigationHeader
+				navigationItems={ [] }
+				title="Plugin updates manager"
+				subtitle="Effortlessly schedule plugin auto-updates with built-in rollback logic."
+			>
+				{ context === 'list' && onCreateNewSchedule && (
```

Comment:
Perhaps you will handle this later, but `Create a new schedule` in the header seems to be:
- not present when number of list items is zero.
- enabled when it's 1
- disabled when it's 2.